templates/ghostty-script-base16.mustache

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env sh
+
+# ===================
+# ghostty-icon-tinter
+# ===================
+
+function _extract_palette_value {
+    # Note: return the last recurrence, in case there are multiple.
+    _theme_file | sed -nE 's/palette = '$1'=(#.*)/\1/p' \
+       | tail -n 1
+}
+
+function _help {
+    2>&1 cat <<EOF
```

Comment:
I don't think this is necessary but I might be wrong.

```suggestion
    cat <<EOF
```

Reply:
I just don't want it to end up in the theme file if things go wrong e.g. invalid flags provided.